From: Egbert Eich <eich@suse.com>
Date: Tue Dec 5 16:27:45 2017 +0100
Subject: Use gnuplot terminal auto-detection
Patch-mainline: N/A
Git-commit: 000d7ff4fcead18a584c822a1e6b564c5f348764
References: 

Don't hard code terminal type for gnuplot.
Updated patch from dmitry_r@opensuse.org.

Signed-off-by: Egbert Eich <eich@suse.com>
---
 src/frontend/plotting/gnuplot.c | 2 ++
 1 file changed, 2 insertions(+)
diff --git a/src/frontend/plotting/gnuplot.c b/src/frontend/plotting/gnuplot.c
index 8444905..196d4c2 100644
--- a/src/frontend/plotting/gnuplot.c
+++ b/src/frontend/plotting/gnuplot.c
@@ -131,11 +131,13 @@ ft_gnuplot(double *xlims, double *ylims, char *filename, char *title, char *xlab
     }
 
     /* Set up the file header. */
+#if 0
 #if !defined(__MINGW32__) && !defined(_MSC_VER)
     fprintf(file, "set terminal X11 noenhanced\n");
 #else
     fprintf(file, "set termoption noenhanced\n");
 #endif
+#endif
     if (title) {
         text = cp_unquote(title);
         fprintf(file, "set title ");
